File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2670.cs
using System;

class URI
{
    static int min(int a, int b){
        if(a > b){
            return b;
        } else{
            return a;
        }
    }
    
    static void Main() {
        int[] a = new int[3];
        int t1, t2, t3;
        a[0] = int.Parse(System.Console.ReadLine().Trim());
        a[1] = int.Parse(System.Console.ReadLine().Trim());
        a[2] = int.Parse(System.Console.ReadLine().Trim());
        
        t1 = a[1]*2+a[2]*4;
        t2 = a[0]*2+a[2]*2;
        t3 = a[0]*4+a[1]*2;
        
        Console.Write("{0}\n", min(min(t1, t2), t3));
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2670.lua
function min(a, b)
   if a > b then
       return b
    else return a end
end

a = {}

a[1] = io.read("*number")
a[2] = io.read("*number")
a[3] = io.read("*number")

t1=a[2]*2+a[3]*4;
t2=a[1]*2+a[3]*2;
t3=a[1]*4+a[2]*2;

print(min(min(t1, t2), t3))